nservicebus/msmq/subscription-persistence_config_core_[5,6].partial.md

```diff
@@ -0,0 +1,11 @@
+WARNING: Storing subscriptions in MSMQ is not suitable when endpoints are scaled-out, because the subscription queue cannot be shared among multiple endpoints.
+
+To configure MSMQ as the subscription persistence:
+
+snippet:ConfiguringMsmqPersistence
+
+By default NServiceBus uses a queue called `NServiceBus.Subscriptions`. If not specified otherwise, all endpoints will share that queue to store subscriptions. In order to specify a different queue, add the following `configSections` and subsequent config entry:
+
+snippet:MsmqSubscriptionAppConfig
+
+WARNING: When using MSMQ Subscription Persistence on multiple endpoints running on the same machine, every endpoint requires a dedicated subscription storage queue.
```

Comment:
Can we remove this warning and make it as part of the explanation in the previous paragraph.